cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,33 @@
-describe('', () => {
-  before(() => {
+const { faker } = require('@faker-js/faker');
 
+describe('Article CRUD', () => {
+  before(() => {
   });
```

Comment:
You've defined a `before` hook here, but it's empty. The `before` hook is usually used to set up the state before running the tests. If you don't need to set up anything before the tests, you can remove this hook.